docker/nginx/nginx.conf

```diff
@@ -41,7 +41,7 @@ http {
 
         location / {
             # try to serve file directly, fallback to app.php
-            try_files $uri /app.php$is_args$args;
+            try_files $uri /app_dev.php$is_args$args;
```

Comment:
Yup, it's my use case. Maybe you have another one for this docker-compose that should be considered?

Same logic that enabling xdebug on php image. Wouldn't have made sense on a production image :-)